StarCellBio/settings.py

```diff
@@ -26,6 +27,12 @@
     # ('Your Name', 'your_email@example.com'),
 )
 
+if S3_BACKEND_ENABLED:
+    DEFAULT_FILE_STORAGE = 'libs.storages.S3Storage.S3Storage'
+    AWS_ACCESS_KEY_ID = ''
+    AWS_SECRET_ACCESS_KEY = ''
+    AWS_STORAGE_BUCKET_NAME = ''
```

Comment:
Rather than setting the `AWS` variables to empty strings it might be better to leave them out and only set the `DEFAULT_FILE_STORAGE` when the s3 variable is set. The other settings will be added in through Ansible and I'm not sure of the order of evaluation between that and the settings here. If left as is these variables may overwrite what I am setting as part of the deployment.